### PR TITLE
Fixed links

### PR DESCRIPTION
**Pull request by alfredmyers**
Up to version [10](https://msdn.microsoft.com/en-us/library/system.diagnostics.eventing.reader(v=vs.100).aspx), the docs used to point to these two other pages, but in version [11](https://msdn.microsoft.com/en-us/library/system.diagnostics.eventing.reader(v=vs.110).aspx) the links where removed leaving some text behind.

>  For more information, see and .

I wasn't able to find the current version of the two referenced docs.

Should the links be added back even if pointing to a previous version of the docs or should the sentence be removed altogether?

### CODE REVIEW

File: xml/ns-System.Diagnostics.Eventing.Reader.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
   <Docs>
-    <summary>Using the <see cref="N:System.Diagnostics.Eventing.Reader" /> namespace, you can develop applications that read and manage event logs. An event in an event log contains information, a warning, or an error that has been published by a specific application, service, or operating system component.  These events are read by applications that monitor a computer's health and applications that take action when specific events occur.  For more information, see  and .</summary>
+    <summary>Using the <see cref="N:System.Diagnostics.Eventing.Reader" /> namespace, you can develop applications that read and manage event logs. An event in an event log contains information, a warning, or an error that has been published by a specific application, service, or operating system component.  These events are read by applications that monitor a computer's health and applications that take action when specific events occur.  For more information, see [Technology Summary for Reading and Managing Event Logs](https://msdn.microsoft.com/en-us/library/bb671205.aspx) and [Event Log Scenarios](https://msdn.microsoft.com/en-us/library/bb671204.aspx).</summary>
```
Comment (rpetrusha) [inline xml/ns-System.Diagnostics.Eventing.Reader.xml]: In the two URLs, can you remove "/en-us"? That links to a non-localized page.

**Review by rpetrusha — DISMISSED** [review]
Thanks for adding the missing links, @alfredmyers. Yes, it's fine to link to topics in an older doc set, since those are the last versions of the topics in our doc set. I also requested a correction in the URLs.

**Review by mairaw — APPROVED** [review]
LGTM

**Comment by mairaw** [comment]
Since you've addressed @rpetrusha comments, I'll go ahead and merge this. Thanks @alfredmyers!

**Pull request merged**